modules/rules_bazel_integration_test/0.12.0/MODULE.bazel

```diff
@@ -0,0 +1,22 @@
+module(
+    name = "rules_bazel_integration_test",
+    version = "0.12.0",
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.4.1")
+bazel_dep(name = "rules_python", version = "0.19.0")
+bazel_dep(name = "platforms", version = "0.0.6")
+bazel_dep(name = "cgrindel_bazel_starlib", version = "0.14.9")
+
+register_toolchains("@bazel_tools//tools/python:autodetecting_toolchain")
```

Comment:
Slightly surprised to see this line, is it needed? I would have assumed the Bazel-provided `bazel_tools` module to register this toolchain automatically.

Reply:
This repo has an example python test driver. If I remove the `register_toolchains("@bazel_tools//tools/python:autodetecting_toolchain")`, I get the following error:

```
ERROR: /Users/chuck/code/bazel-contrib/rules_bazel_integration_test/try_removing_python_toolchain_reg/tests/py_tests/BUILD.bazel:8:27: While resolving toolchains for target //tests/py_tests:test_base_test: No matching toolchains found for types @bazel_tools//tools/python:toolchain_type.
To debug, rerun with --toolchain_resolution_debug='@bazel_tools//tools/python:toolchain_type'
If platforms or toolchains are a new concept for you, we'd encourage reading https://bazel.build/concepts/platforms-intro.
ERROR: Analysis of target '//tests/py_tests:test_base_test' failed; build aborted:
```

Perhaps I am not doing something properly?

I am going to resolve the conversation for now, so that I can get this version into BCR. (I have another PR waiting for this release.) However, I am open to fixing this, if there is a better way.

Reply:
Actually, I can't resolve the conversation. So, let me know how you would like to proceed. Thanks!

Reply:
Interesting, I will take a look, but no reason to block you here.

Reply:
@Wyverald

Reply:
we don't register python automatically (never have, I believe), so this is probably WAI. I would expect `rules_python` to register these, though... @alexeagle do you know what's going on with Python toolchains?